gmplot/gmplot.py

```diff
@@ -28,9 +28,10 @@ def safe_iter(var):
 
 class GoogleMapPlotter(object):
 
-    def __init__(self, center_lat, center_lng, zoom, apikey=''):
+    def __init__(self, center_lat, center_lng, zoom, map_type='', apikey=''):
```

Comment:
should the default be ` google.maps.MapTypeId.ROADMAP` instead of the empty string? this will preserve the current behavior without changing existing code, no?

Reply:
If `map_type`'s an empty string, then we won't explicitly set `mapTypeId` below, so the map type will default to `roadmap` according to [Google Maps](https://developers.google.com/maps/documentation/javascript/maptypes#BasicMapTypes).

As long as Google Maps keeps that as their default, then this code will preserve the current behaviour. If, however, they decide to change their default, then wouldn't a customer prefer that gmplot follows suit and has its default map type match Google Maps'?